src/app/common/crypto-assets/stacks-crypto-asset.utils.ts

```diff
@@ -0,0 +1,47 @@
+import { abbreviateNumber } from '@app/common/utils';
+import { isUndefined } from '@shared/utils';
+import { isValidUrl } from '@shared/utils/validate-url';
+import { StacksFungibleTokenAsset } from '@shared/models/crypto-asset.model';
+
+import { convertUnicodeToAscii } from '../string-utils';
+
+export function removeCommas(amountWithCommas: string) {
+  return amountWithCommas.replace(/,/g, '');
+}
+
+export function getFormattedBalance(amount: string) {
+  const noCommas = removeCommas(amount);
+  const number = noCommas.includes('.') ? parseFloat(noCommas) : parseInt(noCommas);
+  return number > 10000
+    ? {
+        isAbbreviated: true,
+        value: abbreviateNumber(number),
+      }
+    : { value: amount, isAbbreviated: false };
+}
+
+export const isIconUrl = isValidUrl;
```

Comment:
Could you kill this and call isValidUrl directly, that would make much more sense